Cannot install newer version of rxjs then what is specified as peer dependency

I would like to include a newer version than rxjs@5.0.0-beta.6 but npm would not let me:

```npm ERR! peerinvalid The package rxjs@5.0.0-beta.11 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer ng-metadata@2.1.1 wants rxjs@5.0.0-beta.6```

--- a/README.md
+++ b/README.md
@@ -210,7 +210,7 @@ ng-metadata requires certain polyfills in the application environment. We instal
 
 Install peer dependencies by running:
 
-`npm i --save reflect-metadata core-js rxjs@5.0.0-beta.6`
+`npm i --save reflect-metadata core-js rxjs@5.0.0-beta.11`
 
 **core-js** - monkey patches the global context (window) with essential features of ES2015 (ES6). Developers may substitute an alternative polyfill that provides the same core APIs. This dependency should go away once these APIs are implemented by all supported ever-green browsers.
 

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "live-server": "0.8.2",
     "mocha": "2.3.4",
     "reflect-metadata": "0.1.3",
-    "rxjs": "5.0.0-beta.6",
+    "rxjs": "5.0.0-beta.11",
     "sinon": "1.17.2",
     "systemjs": "0.19.6",
     "systemjs-builder": "0.14.11",
@@ -52,7 +52,7 @@
     "validate-commit-msg": "2.0.0"
   },
   "peerDependencies": {
-    "rxjs": "5.0.0-beta.6",
+    "rxjs": "5.0.0-beta.11",
     "angular": ">=1.4.x",
     "reflect-metadata": "0.1.x"
   },

